Gemfile.lock

```diff
@@ -133,19 +133,19 @@ GEM
     docile (1.3.2)
     domain_name (0.5.20190701)
       unf (>= 0.0.5, < 1.0.0)
-    elasticsearch (5.0.5)
-      elasticsearch-api (= 5.0.5)
-      elasticsearch-transport (= 5.0.5)
-    elasticsearch-api (5.0.5)
+    elasticsearch (7.5.0)
```

Comment:
Does `elasticsearch` need to be restricted to `~> 6.0.0` also?

Reply:
Thanks. It was fixed in 6.1, I have removed patch version pinning.